webpack/assets/javascripts/react_app/API.js

```diff
@@ -17,6 +17,18 @@ export default {
       dataType: 'json',
       data: data,
       error: function (jqXHR, textStatus, errorThrown) {
+        /* eslint-disable no-console */
+        console.log(jqXHR);
+      }
+    });
+  },
+  markGroupNotificationAsRead(group) {
+    $.ajax({
+      url: `/notification_recipients/group/${group}`,
+      contentType: 'application/json',
+      type: 'PUT',
+      error: function (jqXHR, textStatus, errorThrown) {
```

Comment:
Nitpick: I think this can be something like `(jqXHR) => {console.log(jqXHR);}`, although I'm not sure if the XHR object is what we want logged here?